_examples/golang-imports/api.gen.go

```diff
@@ -523,6 +552,20 @@ func RequestFromContext(ctx context.Context) *http.Request {
 	r, _ := ctx.Value(HTTPRequestCtxKey).(*http.Request)
 	return r
 }
+
+func MethodFromContext(ctx context.Context) MethodCtx {
```

Comment:
Are we OK to return struct here? Or should we return pointer to the struct?

I guess `struct` copies a little more data.. but on the other hand the returned data is immutable and cannot change the value globally.

Thoughts?

(I'm OK with this as is.)

Reply:
I thought it's better to make it immutable in this case. Therefore  I returned direct value not pointer